### PR TITLE
[Bexley][WW] Couple of minor text changes

### PR DESCRIPTION
**Pull request by chrismytton**
Fixes https://github.com/mysociety/societyworks/issues/4444

Fixes https://github.com/mysociety/societyworks/issues/4450

### CODE REVIEW

**Comment by chrismytton** [comment]
Oops failing test, will look!

**Review by dracos — APPROVED** [review]

**Comment by chrismytton** [comment]
Added an extra commit to take the text in the missed report status section from the config, rather than from the latest update.

Should fix https://github.com/mysociety/societyworks/issues/4444 and https://github.com/mysociety/societyworks/issues/4450

File: perllib/FixMyStreet/Cobrand/Bexley/Waste.pm [hunk 1]
```diff
@@ -463,11 +466,8 @@ sub _missed_collection_reports {
             = $report->get_extra_field_value('service_item_name') // '';
         next if $missed_collection_reports{$service_item_name};
 
-        my $latest_comment
-            = $report->comments->search(
-                {},
-                { order_by => { -desc => 'id' } },
-        )->first;
+        my $status = $ws->{WorksheetStatusName};
```
Comment (dracos) [inline perllib/FixMyStreet/Cobrand/Bexley/Waste.pm:469]: This will always be Open by the if in line 452, that right?

**Review by dracos — COMMENTED** [review]
Don't quite follow, as the linked ticket talks about adding a response template rather than config, but assuming this is what's required.

File: perllib/FixMyStreet/Cobrand/Bexley/Waste.pm (hunk 1 above)
Comment (chrismytton) [reply perllib/FixMyStreet/Cobrand/Bexley/Waste.pm]: No, that was not right! Clearly wasn't thinking straight yesterday.

Comment (chrismytton) [reply perllib/FixMyStreet/Cobrand/Bexley/Waste.pm]: Have pushed a fixup commit which uses the fms_state matched to the report->state to determine which text to display.

File: perllib/FixMyStreet/Cobrand/Bexley/Waste.pm [hunk 2]
```diff
@@ -446,6 +446,11 @@ sub _missed_collection_reports {
     my $config = $self->feature('whitespace');
     my $state_mapping = $config->{missed_collection_state_mapping};
 
+    # Create a lookup hash mapping fms_state to text
```
Comment (dracos) [inline perllib/FixMyStreet/Cobrand/Bexley/Waste.pm:449]: Can we assume there won't be more than one thing with the same `fms_state` (or that it'll pick one at random if there is)?

File: perllib/FixMyStreet/Cobrand/Bexley/Waste.pm [hunk 3]
```diff
@@ -446,6 +446,11 @@ sub _missed_collection_reports {
...
+    my %fms_state_to_text = map {
```
Comment (dracos) [inline perllib/FixMyStreet/Cobrand/Bexley/Waste.pm:450]: As you don't need the keys, you could have `map { $_->{fms_state} => $_->{text} } values %$state_mapping` here.

**Review by dracos — COMMENTED** [review]
Looks fine, assuming we don't mind about the possibility of duplicate

File: perllib/FixMyStreet/Cobrand/Bexley/Waste.pm (hunk 2 above)
Comment (chrismytton) [reply perllib/FixMyStreet/Cobrand/Bexley/Waste.pm]: Good point, there are lots of things for the same fms state, so this isn't going to work.

**Comment by chrismytton** [comment]
Have abandoned the code changes for pulling the status text from config. Just going with an amended text change for now.

**Pull request merged**